test/e2e/provisioning/all_e2e_test.go

```diff
@@ -128,9 +128,11 @@ func TestOpenstackProvisioningE2E(t *testing.T) {
 	osUsername := os.Getenv("OS_USERNAME")
 	osTenant := os.Getenv("OS_TENANT_NAME")
 	osNetwork := os.Getenv("OS_NETWORK_NAME")
+	osInstanceReadyCheckPeriod := os.Getenv("INSTANCE_READY_CHECK_PERIOD")
```

Comment:
Why INSTANCE_READY_CHECK_PERIOD and INSTANCE_READY_TIMEOUT are is read from the env vars? I would suggest that you simply set them in the template:
https://github.com/kubermatic/machine-controller/blob/master/test/e2e/provisioning/testdata/machinedeployment-openstack.yaml#L26-L36